.github/workflows/continuous-integration.yml

```diff
@@ -21,16 +21,16 @@ jobs:
             epVersion: 2.4.0
           - os: macos-latest
             java: 11
-            epVersion: 2.20.0
+            epVersion: 2.21.0
```

Comment:
Seeing this change, I wonder if there is a way to define this version as a constant once in this yml and reference it in all these jobs and the check below, like we do inside gradle :)

Reply:
Unfortunately this is not possible.  You can define variables in an `env` context but you can't use them when specifying properties in the `matrix`, see [here](https://github.com/orgs/community/discussions/26388).